libs/api/domains/consultation-portal/src/lib/cases/cases.service.ts

```diff
@@ -0,0 +1,72 @@
+import { Inject, Injectable } from '@nestjs/common'
+import { ApolloError } from 'apollo-server-express'
+import { FetchError } from '@island.is/clients/middlewares'
+import {
+  CasesApi,
+  ApiCasesCaseIdAdvicesGetRequest,
+  ApiCasesCaseIdAdvicesPostRequest,
+  ApiCasesGetRequest,
+  ApiCasesCaseIdGetRequest,
+} from '@island.is/clients/consultation-portal'
+
+import { AuthMiddleware } from '@island.is/auth-nest-tools'
+import type { Auth, User } from '@island.is/auth-nest-tools'
+import type { Logger } from '@island.is/logging'
+import { LOGGER_PROVIDER } from '@island.is/logging'
+import { CaseResult } from '../models/caseResult.model'
+import { CaseItemResult } from '../models/caseItemResult.model'
+@Injectable()
+export class CaseResultService {
+  constructor(
+    @Inject(LOGGER_PROVIDER)
+    private logger: Logger,
+    private casesApi: CasesApi,
+  ) {}
+  handleError(error: FetchError | ApolloError): void {
+    this.logger.error(error)
+
+    throw new ApolloError(
+      'Failed to resolve request',
+      error?.message ?? error?.response?.message,
+    )
+  }
```

Comment:
This is not necessary. Default error handling is pretty good with EnhancedFetch.